test/tools/ossfuzz/abiV2Fuzzer.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <fstream>
+#include <test/tools/ossfuzz/abiV2FuzzerCommon.h>
+
+using namespace dev::test::abiv2fuzzer;
+using namespace std;
+using namespace dev;
+
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
+{
+	// Solidity program
+	string code(reinterpret_cast<char const*>(_data), _size);
+	// Solidity runtime bytecode
+	dev::bytes byteCode;
+	// Hex encoded runtime bytecode
+	std::string hexEncodedRuntimeCode;
```

Comment:
Does this need to be declared here or can you just move it to the definition?

Reply:
done